stone-webapi-breakeven/Controllers/AccountBankingController.cs

```diff
@@ -41,7 +41,7 @@ public IActionResult CreateAccountBanking([FromBody] AccountBankingDto accountBa
         public IActionResult GetAccountBankingById(int id)
         {
 
-            var result = _context.AccountsBanking.FirstOrDefault(accountBanking => accountBanking.AccountBankingId == id);
+            var result = _service.GetAccountBankingById(id);
             if (result == null) return NotFound();
```

Comment:
totalmente opcional.

```suggestion
            var AccountBanking = _service.GetAccountBankingById(id);

            if (result is null)
            {
                 return NotFound();
            }
```

Reply:
Resolvido no commit: f46357004f3e0957a06ae589d3a6996b8ca711f2